src/loot_filter.py

```diff
@@ -82,14 +82,12 @@ def check_items(inv: InventoryBase):
         res = Filter().should_keep(item_descr)
         matched_any_affixes = len(res.matched) > 0 and len(res.matched[0].matched_affixes) > 0
         Logger.debug(f"  Runtime (Filter): {time.time() - start_filter:.2f}s")
-        if not res.keep:
-            keyboard.send("space")
-            wait(0.13, 0.14)
-        elif res.keep and (matched_any_affixes or item_descr.rarity == ItemRarity.Unique):
+        # mark everything as junk
```

Comment:
Items that match affixes will be marked as favorite. Items that match aspects will just be kept. Everything else will be marked as junk.

If we do this the items that only match the aspect will also be junked.

Reply:
Yup you are correct. Will fix